demos/interactive_face_detection_demo/cpp/interactive_face_detection.hpp

```diff
@@ -13,47 +13,63 @@
 DEFINE_INPUT_FLAGS
 DEFINE_OUTPUT_FLAGS
```

Comment:
I think we should replace these macros with actual vars, that way it will be easier to see the whole list of flags for demo